tapfreighter/chain_porter.go

```diff
@@ -931,7 +931,6 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 			ctx, &AnchorVTxnsParams{
 				FeeRate:            feeRate,
 				VPkts:              []*tappsbt.VPacket{vPacket},
-				InputCommitments:   currentPkg.InputCommitments,
```

Comment:
Re the commit message: IIRC we ended up special casing the passive assets so we wouldn't have to force a dev/caller to handle all that book keeping logic. With this new approach, will the freighter automatically create the relevant vPSBTs for the passive assets, or is that something the caller needs to be burdened with?

Reply:
Yes, they will still be created automatically. If you use the `FundVirtualPsbt -> AnchorVirtualPsbts` flow you don't have to deal with passive assets at all. If you use the `FundVirtualPsbt -> CommitVirtualPsbts -> PublishAndLogTransfer` flow (implemented in part 3 of the saga), you'll need to carry along the passive assets and sign them manually (since they might also be multisig or custom spend scripts so you need to do manual signing anyway) and feed them into the calls.